src/components/Button/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
import './styles.scss` instead of import 'components/Button/style.scss';

```
import PropTypes from 'prop-types';

import 'components/Button/style.scss';
import classNames from 'classnames';
```
=>
```
import React from 'react'
import PropTypes from 'prop-types';
import classNames from 'classnames';

import './styles.scss';
```

Please do not forget importing `React`